api/libproject/devstart.rb

```diff
@@ -1348,3 +1348,25 @@ def setup_cloud_project(cmd_name, *args)
   :description => "Initializes resources within a cloud project that has already been created",
   :fn => lambda { |*args| setup_cloud_project("setup-cloud-project", *args) }
 })
+
+def start_api_and_incremental_build(cmd_name, args)
```

Comment:
Can you document where the logs go in our README?

Reply:
This line from the output does tell you where to look:
```
2018-03-28 15:05:53.841:INFO:oejs.Server:main: Started @18203ms
api_1                          | Dev App Server output written to : /w/api/build/dev-appserver-out/dev_appserver.out
```
I could also put that in the README, but then it would be misleading if that location changes.

Reply:
READMEs are always at risk of being out of date, and this particular thing seems unlikely to change. It is easy to overlook log statements like this.

If you like, the README can say "check the console output for where log output goes to, normally api/build/dev-appserver-out and public-api/build/dev-appserver-out." I'd like to have something in there indicating where our logs wind up.

Reply:
This doesn't apply anymore since the logs appear directly in the console.